src/styles/header/_index.scss

```diff
@@ -17,52 +17,6 @@
     text-overflow: ellipsis;
   }
 
-  .#{$lu_css_prefix}-sort {
-    @include fa-icon();
-
-    opacity: 0;
-
-    &[data-sort=asc]::before {
```

Comment:
how is the sorting order encoded instead?

Reply:
I don't know exactly what you mean. I defined the icons for the sort order in the _toolbar.scss.
https://github.com/Caleydo/lineupjs/blob/thinkh/header-redesign/src/styles/header/_toolbar.scss#L128

Reply:
I meant what is the status of encoding of the sorting hierarchy. i.e. which is the first sorting criteria, the second, ...